FlowCrypt/Functionality/Mail Provider/Imap/Imap+retry.swift

```diff
@@ -7,117 +7,118 @@
 //
 
 import Foundation
-import Promises
+import MailCore
 
 extension Imap {
-    func finalize<T>(
+
+    func executeVoid(
```

Comment:
`execute` and `executeVoid` could probably be united somehow. But my attempt was ugly. Could do in another issue.